path_config_test.go

```diff
@@ -535,6 +535,65 @@ func TestConfig_LocalJWTRenewal(t *testing.T) {
 	}
 }
 
+func TestConfig_SystemCa(t *testing.T) {
+	testCases := map[string]struct {
+		config              map[string]interface{}
+		setupInClusterFiles bool
+		expected            *kubeConfig
+	}{
+		"no CA default to system": {
+			config: map[string]interface{}{
+				"kubernetes_host":      "host",
+				"disable_local_ca_jwt": false,
+				"kubernetes_ca_cert":   "",
+			},
+			setupInClusterFiles: true,
+
+			expected: &kubeConfig{
+				PublicKeys:           []interface{}{},
+				PEMKeys:              []string{},
+				Host:                 "host",
+				CACert:               testLocalCACert,
+				TokenReviewerJWT:     testLocalJWT,
+				DisableISSValidation: true,
+				DisableLocalCAJwt:    false,
+			},
+		},
```

Comment:
Is this test different than "no CA or JWT, default to local" in `TestConfig_LocalCaJWT()`? IIRC `disable_local_ca_jwt` defaults to false, and `kubernetes_ca_cert` defaults to `""`.
https://github.com/hashicorp/vault-plugin-auth-kubernetes/blob/a0dfafee6502d974689c2ce60a30265583d37d1f/path_config_test.go#L361-L375